precompiles/staking/testdata/StakingCaller.sol

```diff
@@ -334,15 +332,12 @@ contract StakingCaller {
             (shares, coin) = abi.decode(data, (uint256, staking.Coin));
         } else if (calltypeHash == keccak256(abi.encodePacked("callcode"))) {
             //Function signature
-            bytes4 sig = bytes4(keccak256(bytes("delegation(address,string)")));
+            bytes4 sig = bytes4(keccak256(bytes("delegation(address,address)")));
```

Comment:
We need @GAtom22 to verify this change, I don't enough of solidity to be sure that it's the correct change (it looks good)